docs/security.md

```diff
@@ -76,6 +76,36 @@ function transferFrom(address who, address to, uint amount)
 }
 ```
 
+## Speed Bump
+
+If we would like to prevent off-chain calls from being chained together, we can
+ensure that the block has been finalized.
+
+```solidity
+contract Secret {
+  uint256 private _height;
+  bytes private _secret;
+  address private _buyer;
+
+  constructor(bytes memory _text) {
+    _secret = _text;
```

Comment:
Should also initialize `_height` to some maximum value or have an additional flag (maybe `_buyer` being non-equal to `address(0)`) as otherwise `_height` is initialized to zero and every block number will be greater than that.

Reply:
Makes sense. Opting to check `_buyer`.